css/portal-dashboard/student-navigator.less

```diff
@@ -77,7 +74,8 @@
     }
   }
 
-  .studentAreaFiller {
-    height: 200px;
+  &.filler {
+    background-color: @cc-orange-light6;
+    height: 250px;
```

Comment:
I think this is better as part of a div somewhere else.  Doesn't really seem like part of the student navigator.